playground/mocks/config/responseConfig.js

```diff
@@ -181,6 +183,9 @@ const idx = {
     'error-authenticator-phone-voice-ratelimit',
   ],
   '/idp/idx/enroll/new': [
+    //'authenticator-enroll-email',
+    //'authenticator-enroll-email-emailmagiclink-true',
+    //'authenticator-enroll-email-emailmagiclink-false',
     'error-new-signup-email',
     'error-new-signup-email-exists'
```

Comment:
nit: maybe move the new additions under the existing files, just to signify these aren't the default responses

Reply:
what do you mean existing files?

Reply:
Just suggesting to keep 
```
'error-new-signup-email',
'error-new-signup-email-exists'
```
at the top of the array here, and move the commented lines under

Reply:
Done